apps/ui/src/networks/offchain/api/queries.ts

```diff
@@ -166,8 +166,8 @@ export const SPACE_QUERY = gql`
 `;
 
 export const USER_VOTES_QUERY = gql`
-  query ($spaceIds: [String], $voter: String) {
-    votes(where: { space_in: $spaceIds, voter: $voter }) {
+  query ($first: Int, $spaceIds: [String], $voter: String) {
+    votes(first: $first, where: { space_in: $spaceIds, voter: $voter }) {
```

Comment:
Is there a reason why we don't add `$skip` there? In common API we don't have `first` and `skip` at all.